vmm/src/vm.rs

```diff
@@ -2439,6 +2439,7 @@ impl Vm {
 
 impl Pausable for Vm {
     fn pause(&mut self) -> std::result::Result<(), MigratableError> {
+        info!("VM has been paused");
```

Comment:
IMHO here is not the right place for this and contradicts just the line below, would line 2476 and lines after be more appropriate 🧐